src/content/docs/faq.md

```diff
@@ -413,3 +413,17 @@ With `stderr`:
 
 - No special setup necessary in order to run in a pipe command
 - Unconventional and that might subvert users expectations
+
+Out of the box, `stdout` will be faster than `stderr` because it is buffered. However you can very
+easily make `stderr` buffered too by wrapping it in a [`BufWriter`] like so:
+
+```rust
+let mut terminal = Terminal::new(CrosstermBackend::new(std::io::stderr()))?;
```

Comment:
```suggestion
let mut terminal = Terminal::new(CrosstermBackend::new(BufWriter::new(std::io::stderr())))?;
```

Reply:
Right I forgot the most important :sweat_smile: , fixed